api.json

```diff
@@ -1311,6 +1312,22 @@
          "mempool_coins": {
            "type":"boolean",
            "description":"Any Rosetta implementation that can update an AccountIdentifier's unspent coins based on the contents of the mempool should populate this field as true. If false, requests to `/account/coins` that set `include_mempool` as true will be automatically rejected."
+          },
+         "block_hash_case": {
+           "description":"This specifies the normalized case for block hash in the BlockIdentifier. If not specified, it's assumed to be case sensitive.",
```

Comment:
nit: drop the the before BlockIdentifier in this and other descriptions. 
`"This specifies the normalized case for block hash in BlockIdentifier.` or `This specifies the normalized case for a block hash in a BlockIdentifier.`

Same comment for `transaction_hash_case` descriptions. :)